wallet-new/src/Cardano/Wallet/API/Request/Filter.hs

```diff
@@ -349,6 +349,7 @@ instance
       where
         incorporate =
             foldr (uncurry appendToQueryString) req . toQueryString
+    hoistClientMonad _ _ _ _ = error "no hoistClientMonad here"
```

Comment:
@parsonsmatt I'm pretty sure this is wrong. Do you happen to know what this should be?

haddocks at http://hackage.haskell.org/package/servant-client-core-0.14.1/docs/Servant-Client-Core-Internal-HasClient.html#v:hoistClientMonad

Reply:
as above, I'd use the ones form the query params:

```haskell
  hoistClientMonad pm _ f cl = \as ->
    hoistClientMonad pm (Proxy :: Proxy api) f (cl as)
```